src/NServiceBus.Core/Routing/RoutingFeature.cs

```diff
@@ -47,7 +47,7 @@ protected internal override void Setup(FeatureConfigurationContext context)
                 EnableBestPracticeEnforcement(context);
             }
 
-            unicastBusConfig?.MessageEndpointMappings.Apply(publishers, unicastRoutingTable, transportInfrastructure.MakeCanonicalForm, conventions);
+            unicastBusConfig?.MessageEndpointMappings.Apply(publishers, unicastRoutingTable, transportInfrastructure.MakeCanonicalForm, conventions, context.Settings.GetAvailableTypes());
```

Comment:
@SzymonPobiega see https://github.com/Particular/NServiceBus/pull/4097 locally the tests were green :fireworks: